src/Solver.jl

```diff
@@ -232,12 +232,13 @@ function writeOutput(f_int, gamma, Va, Vb, Vc, obs, Lam, Par)
     end
     println(")")
     function givefreqs()
+        N < 7 && return 1, 2, 3
```

Comment:
Obviously this should be `N <= 7`, right? for N=7, f2 would be 0, still causing causing out of bonds error at line 240

Reply:
but anyway I suppose there is no harm in including N=7

Reply:
I totally agree we should not waste more of our time here. 
With this fix I will be able to create smaller regression test cases as we discussed, and it's enough for me.